src/PortToTripleSlash/src/libraries/ToTripleSlashPorter.cs

```diff
@@ -91,7 +91,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
             cancellationToken.ThrowIfCancellationRequested();
 
             CollectFiles();
-            if (!_docsComments.Types.Any())
+            if (_docsComments.Types.Count == 0)
```

Comment:
is it possible to use `!_docsComments.Types.HasItems()` instead?

Reply:
CA1860 recommends using Count for perf and readability purposes.

Reply:
Not sure if `!HasItems()` is more performant than `Count == 0`. I applied the Roslyn suggestion.